test/CMakeLists.txt

```diff
@@ -434,6 +467,10 @@ _Block_release(void) { }\n")
             PROPERTY FOLDER "Tests/check-swift")
       endforeach()
     endforeach()
+
+    write_component_test_script("${test_bin_dir}" "irgen" "IRGen" )
+    write_component_test_script("${test_bin_dir}" "silgen" "SILGen")
+    write_component_test_script("${test_bin_dir}" "typecheck" "Sema")
```

Comment:
Nit: `sema` please. We do a lot more than typechecking in semantic analysis.

Reply:
Done.